pkg/services/accesscontrol/acimpl/accesscontrol.go

```diff
@@ -92,6 +96,10 @@ func (a *AccessControl) evaluateZanzana(ctx context.Context, user identity.Reque
 	}
 
 	return eval.EvaluateCustom(func(action, scope string) (bool, error) {
+		if implemented := zanzana.CheckAvailableAction(action); !implemented {
+			return false, ErrAccessNotImplemented
+		}
```

Comment:
Not sure if we need `CheckAvailableAction`, translation should fail a couple of lines down and we can just return the error there then